integration/mediation-tests/tests-service/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/endpoints/wsdlEpTest.xml

```diff
@@ -0,0 +1,4059 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Unless this is a file move, the license header is wrong

Reply:
changed to 2019.